### ISSUE TITLE
🐛 Bug Report: Error: Could not fetch catalog entities.

### ISSUE THREAD

**Issue by ankit-arora-369** [backstage/backstage#15345]
### 📜 Description

I am trying to setup backstage on ubuntu22 using the following link:
**https://backstage.io/docs/getting-started/**

Default setup is breaking and I am getting following CORS and other errors without any changes made to app-config.yaml file:

```
Access to fetch at 'http://localhost:7007/api/catalog/entity-facets?facet=kind' from origin 'http://127.0.0.1:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

WebSocket connection to 'ws://127.0.0.1:3000/ws' failed: 
```

The command used to create default app is as follows:

**npx @backstage/create-app**

![bs](https://user-images.githubusercontent.com/27761451/208744383-977da18a-8e22-426a-b78c-76caad7b6e08.png)

### 👍 Expected behavior

It should be running without any error coming and should be showing example apps.

### 👎 Actual Behavior with Screenshots

Throwing CORS and other errors.

### 👟 Reproduction steps

The command used to create default app is as follows:

**npx @backstage/create-app**

**cd  <app_name> && yarn dev**

Then open URL 127.0.0.1:3000 in browser.

### 🖥️ Your Environment

OS:   Linux 5.10.16.3-microsoft-standard-WSL2 - linux/x64
node: v16.19.0
yarn: 1.22.19
cli:  0.22.0 (installed)
backstage:  1.9.0

Dependencies:
  @backstage/app-defaults                          1.0.9
  @backstage/backend-common                        0.17.0
  @backstage/backend-plugin-api                    0.2.0
  @backstage/backend-tasks                         0.4.0
  @backstage/catalog-client                        1.2.0
  @backstage/catalog-model                         1.1.4
  @backstage/cli-common                            0.1.11
  @backstage/cli                                   0.22.0
  @backstage/config-loader                         1.1.7
  @backstage/config                                1.0.5
  @backstage/core-app-api                          1.3.0
  @backstage/core-components                       0.12.1
  @backstage/core-plugin-api                       1.2.0
  @backstage/errors                                1.1.4
  @backstage/integration-aws-node                  0.1.0
  @backstage/integration-react                     1.1.7
  @backstage/integration                           1.4.1
  @backstage/plugin-api-docs                       0.8.12
  @backstage/plugin-app-backend                    0.3.39
  @backstage/plugin-auth-backend                   0.17.2
  @backstage/plugin-auth-node                      0.2.8
  @backstage/plugin-catalog-backend                1.6.0
  @backstage/plugin-catalog-common                 1.0.9
  @backstage/plugin-catalog-graph                  0.2.24
  @backstage/plugin-catalog-import                 0.9.2
  @backstage/plugin-catalog-node                   1.3.0
  @backstage/plugin-catalog-react                  1.2.2
  @backstage/plugin-catalog                        1.7.0
  @backstage/plugin-github-actions                 0.5.12
  @backstage/plugin-org                            0.6.2
  @backstage/plugin-permission-common              0.7.2
  @backstage/plugin-permission-node                0.7.2
  @backstage/plugin-permission-react               0.4.8
  @backstage/plugin-proxy-backend                  0.2.33
  @backstage/plugin-scaffolder-backend             1.9.0
  @backstage/plugin-scaffolder-common              1.2.3
  @backstage/plugin-scaffolder                     1.9.0
  @backstage/plugin-search-backend-module-pg       0.5.0
  @backstage/plugin-search-backend-node            1.1.0
  @backstage/plugin-search-backend                 1.2.0
  @backstage/plugin-search-common                  1.2.0
  @backstage/plugin-search-react                   1.3.0
  @backstage/plugin-search                         1.0.5
  @backstage/plugin-tech-radar                     0.5.19
  @backstage/plugin-techdocs-backend               1.5.0
  @backstage/plugin-techdocs-module-addons-contrib 1.0.7
  @backstage/plugin-techdocs-node                  1.4.3
  @backstage/plugin-techdocs-react                 1.1.0
  @backstage/plugin-techdocs                       1.4.1
  @backstage/plugin-user-settings                  0.6.0
  @backstage/release-manifests                     0.0.8
  @backstage/test-utils                            1.2.3
  @backstage/theme                                 0.2.16
  @backstage/types                                 1.0.2
  @backstage/version-bridge                        1.0.3
Done in 0.99s.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by ankit-arora-369** [comment]
One more thing to notice is that, after executing command "npx @backstage/create-app", it's not asking me to choose which DB option to choose. I want to work with Postgres.

**Comment by ankit-arora-369** [comment]
My bad, it worked perfectly fine in browser when I used "localhost:3000" instead of "127.0.0.1:3000".

But any idea why it didn't ask for DB options?

**Comment by freben** [comment]
Yeah, that option was removed. It just made the templates complex, and besides, you typically want to use different setups for local dev and production anyway. So what we generate for you, is a [base config that has sqlite](https://github.com/backstage/backstage/blob/415cd3245064d10e63688fd3c5cf8601893205e1/packages/create-app/templates/default-app/app-config.yaml.hbs#L30), and a [production config that has postgres](https://github.com/backstage/backstage/blob/415cd3245064d10e63688fd3c5cf8601893205e1/packages/create-app/templates/default-app/app-config.production.yaml#L17).

I'll close this as fixed!
